packages/lib/src/sse-util.ts

```diff
@@ -8,3 +8,10 @@ export const sseSignals: NodeJS.Signals[] = ['SIGTERM', 'SIGINT', 'SIGHUP', 'SIG
 export function handleSignals(listener: NodeJS.SignalsListener): void {
 	sseSignals.forEach(signal => process.on(signal, listener))
 }
+
+/**
+ * error Event from eventsource doesn't always overlap with MessageEvent
+ *
+ * TODO: update DefinitelyTyped https://github.com/DefinitelyTyped/DefinitelyTyped/tree/master/types/eventsource
+ */
+export type EventSourceError = MessageEvent & Partial<{ status: number; message: string }>
```

Comment:
Is there a reason you used `Partial` on that in-line type rather than just `{ status?: number; message?: string }`?

Reply:
No, I will update it to your suggestion. Thanks.